src/pages/AnnualReport/AnnualReportTable/AnnualReportInformation/AnnualReportInformation.less

```diff
@@ -1,29 +1,28 @@
 .annualreport-modal {
-    .textCenter {
-        text-align: center;
-    }
-    
-    .container {
-        width: 100%;
-    }
+  .textCenter {
```

Comment:
Why do you change tabulation from 4 spaces to 2?

Reply:
I use Visual Studio Code extension "Prettier - Code formatter", it changed tabulation. Is it bad practise to do so (use 2 spaces for tabulation) and i should rewrite it back to 4 spaces?

Reply:
As i remember, there is no any format rules for tabulation on EPlast, but you don't need to change it in every file. You can leave it with 2 spaces, but it is just some useless work to change the tabulation

Reply:
Ok, thanks